packages/transport/src/modules/sync/ChangedItems.ts

```diff
@@ -12,7 +12,8 @@ export class ChangedItems implements IChangedItems {
     public constructor(
         public readonly relationships: Relationship[] = [],
         public readonly messages: Message[] = [],
-        public readonly identityDeletionProcesses: IdentityDeletionProcess[] = []
+        public readonly identityDeletionProcesses: IdentityDeletionProcess[] = [],
+        public readonly changedObjectIdentifiersDuringDatawalletSync: string[] = []
```

Comment:
Do we really need this in this form and in this place? It looks a little out of place among the `relationships`, `messages` and `identityDeletionProcesses` arrays. Isn't it possible, for example, to obtain the `id` of a changed `identityDeletionProcess` via the `identityDeletionProcesses` array?

Reply:
the other ones are for the external event syncs, the new one tracks changes during datawallet sync.

Reply:
The goal of exactly this implementation is, that we can easily add new events for other thing that are changes - without touching the Datawallet / Sync itself again, only the AccountController.